docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<p>I think it would make sense to compare RS and FS cells for optotagging.</p><p>The hypotheses is that PV+ cells (responsive to opt-tagging ) should also be FS cells. You should state it and confirm.</p><p> </p><p>I would make this comparison much more apparent. Perhaps showing the two plots side by side. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>